lib/plugins/features/movies/controller.js

```diff
@@ -1,9 +1,29 @@
 'use strict';
 
 const Movie = require('../../../models/movie');
+const Movies = Movie.collection();
 
 exports.create = async (payload) => {
   const movie = await new Movie().save(payload);
 
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.find = async (query) => {
+  const movies = Movies.query((queryBuilder) => {
```

Comment:
Instead of calling this on the collection, usually we just create a new model like so:

`new Movie().query(...)`

That way you don't have to use `Movie.collection()` directly.

Also, since we're not doing any extra work after fetching the movies, you can just return here instead of assigning the result to a variable and then returning a variable.